src/MSBuild.UnitTests/XMake_Tests.cs

```diff
@@ -2711,6 +2711,13 @@ public void EndToEndMinimumMessageImportance(string arguments, MessageImportance
 
             TransientTestProjectWithFiles testProject = testEnvironment.CreateTestProjectWithFiles(projectContents);
 
+            // If /bl is specified, set a path for the binlog that is defined by the test environment
+            if (arguments.Contains("/bl"))
+            {
+                string binlogPath = Path.Combine(testProject.TestRoot, "output.binlog");
```

Comment:
Could we consider having a per-verbosity binlog? Meaning make this like Path.Combine(testProject.TestRoot, "diagoutput.binlog") for diag?

Reply:
Updated